acceptance/tests/facts/verify_facts_match_schema.rb

```diff
@@ -0,0 +1,111 @@
+test_name 'C62454: verify facts return from facter match the schema' do
```

Comment:
I would argue that this is a set of unit tests that should be covered at that layer and do not belong in an acceptance test.